TransformationSystem/Utilities/ReplicationTransformation.py

```diff
@@ -0,0 +1,94 @@
+"""
+Utilities to create replication transformations
+"""
+
+from DIRAC.TransformationSystem.Client.Transformation import Transformation
+from DIRAC.TransformationSystem.Client.TransformationClient import TransformationClient
+from DIRAC import gLogger, S_OK, S_ERROR
+
+
+def createDataTransformation(transformationType, targetSE, sourceSE,
+                             metaKey, metaValue,
+                             extraData=None, extraname='',
+                             groupSize=1,
+                             plugin='Broadcast',
+                             enable=False,
+                             ):
+  """Creates the replication transformation based on the given parameters
```

Comment:
Is also "Moving" transformation possible?

Reply:
Yes, moving is also possible.
There is a misuse of the term TransformationType inside the script, because the "TransfomrationType" is always Replication. Though, there are different "flavours" just replication, or replication followed by removal. Which is also called tranformationType inside the script

Reply:
I renamed the type to flavour